docs/machine-learning/how-to-guides/load-data-ml-net.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Load data from files and other sources
 description: This how-to shows you how to load data for processing and training into ML.NET. The data is originally stored in files or other data sources such as databases, JSON, XML or in-memory collections.
-ms.date: 08/01/2019
```

Comment:
Did we forget author here?

Reply:
Couldn't think of a clever pen name. Fixed 🙂